src/walltime.cc

```diff
@@ -213,15 +234,27 @@ std::string DateTimeString(bool local) {
   typedef std::chrono::system_clock Clock;
   std::time_t now = Clock::to_time_t(Clock::now());
   char storage[128];
+  std::size_t written;
 
-  std::tm timeinfo;
-  std::memset(&timeinfo, 0, sizeof(std::tm));
   if (local) {
-    localtime_r(&now, &timeinfo);
+#if defined(OS_WINDOWS)
```

Comment:
Since you don't express a firm opinion and mine is that swapping the conditionals will make the code exactly as unclear ;) I will leave it as it is and in your hands ;)